bap-builder/PackageMode.go

```
@@ -454,11 +517,39 @@ func checkSysrootDirs(platformString *bringauto_package.PlatformString) (error)
+		return false, err
+	}
+	configList, err := prepareConfigs(packageJsonPaths)
+	if err != nil {
+		return false, err
+	}
+
+	sysroot := bringauto_sysroot.Sysroot{
+		IsDebug:        false,
+		PlatformString: platformString,
+	}
+	err = bringauto_prerequisites.Initialize(&sysroot)
```

golangci-lint (1.62.2)

ineffectual assignment to err

(ineffassign)